src/ProfessionalGraveyard.tsx

```diff
@@ -63,12 +103,12 @@ export function ProfessionalGraveyard(props: GraveyardProps) {
                     alt='Fire'
                 />
             </div>
-
             <span className="uppercase text-2xl">
                 RIP you slimey bastards
             </span>
-
-            {data}
+            <button onClick={handleToggleFilter}>Filter ▼</button>
```

Comment:
If a set of tags has a value contained within them (The `Filter ▼`) I like it to be brought to a new line like the span above